src/hmem_rocr.c

```diff
@@ -183,13 +203,18 @@ static int rocr_memcpy(void *dest, const void *src, size_t size)
 	return -FI_EIO;
 }
 
-static int rocr_host_memory_ptr(void *host_ptr, void **ptr)
+static int rocr_host_memory_ptr(void *host_ptr, void **ptr,
+								hsa_agent_t *agent, size_t *size,
+								uint64_t *offset, bool *system)
```

Comment:
indentation is wonky

Reply:
The wonkiness in the display looks to be an issue with how github displays tab. I have VI set to have tab==4 spaces. But I think the default tab display in here is tab==8spaces.
If you add "?ts=4" in the url, it displays properly.

Reply:
oh. didn't know that. Will adjust.